src/test/libcephfs/test.cc

```diff
@@ -3691,3 +3691,62 @@ TEST(LibCephFS, SnapdirAttrs) {
   ASSERT_EQ(0, ceph_unmount(cmount));
   ceph_shutdown(cmount);
 }
+
+TEST(LibCephFS, SnapdirAttrsOnSnapOperations) {
```

Comment:
Maybe we want to split the tests for create, rename, and rm snap operations?

Reply:
ACK :+1: